.ansible-lint

```diff
@@ -5,3 +5,6 @@ warn_list:
   - galaxy[tags]
   - no-handler
   - var-naming[no-role-prefix]
+
+exclude_paths:
+  - .ansible  # where Molecule / Ansible installs dependencies in CI
```

Comment:
not sure why, but linting started [failing in CI](https://github.com/UCL-MIRSG/ansible-collection-infra/actions/runs/13245334270/job/36970342657?pr=164#step:2:508) because of lint issues with some of our dependencies:
>[indentation]: Wrong indentation: expected at least 1
>.ansible/collections/ansible_collections/community/library_inventory_filtering_v1/changelogs/config.yaml:15

This doesn't happen locally. Locally, there is a `.ansible` folder created when I run pre-commit, but it only contains `mirsg.infrastructure` and not the dependencies